src/ImapDump/Command/AuthCommand.php

```diff
@@ -23,15 +23,32 @@ protected function execute(InputInterface $input, OutputInterface $output)
         $host     = $input->getArgument('host');
         $username = $input->getArgument('username');
 
-        $question = new Question('Password: ');
-        $question->setHidden(true);
+        if (!$password = $this->checkpass($username)) {
+            $question = new Question('Password: ');
+            $question->setHidden(true);
 
-        $questionHelper = new QuestionHelper();
-        $password = $questionHelper->ask($input, $output, $question);
+            $questionHelper = new QuestionHelper();
+            $password = $questionHelper->ask($input, $output, $question);
+        }
 
         $server = new Server($host);
         $connection = $server->authenticate($username, $password);
 
         return $connection;
     }
+
+    protected function checkpass(string $username)
+    {
+        $configfile = getcwd().'/config.ini';
+
+        if (file_exists($configfile)) {
+            $config = parse_ini_file($configfile, true);
+
+            if (!empty($config['accounts'][$username])) {
```

Comment:
Please check if the index exists in the $config array using the array_key_exists(..) function before working with it.